src/components/chips/chips.js

```diff
@@ -28,12 +24,24 @@ const chipItem = css`
   transform: translateZ(0.1);
 
   span {
-    color: #1A1A1A;
+    color: var(--color-teal30);
     margin-left: 10px;
   }
 
-  &.active {
-    border: 1px solid #000;
+  &.active, &:hover {
+    border: 1px solid var(--color-grey10);
+  }
+
+  .pocket-theme-dark & {
+    color: var(--color-white100);
+
+    span {
+      color: var(--color-white100);
+    }
+
+    &.active, &:hover {
+      border-color: var(--color-white100);
+    }
```

Comment:
I think we could potentially add more action/usage based color variables or edit the current ones so we won't have to do as much checking for the dark mode class but it's 100% not necessary right now